File: src/com/owlrobotics/pedropathingvisualizer/pedropathing/util/FieldRotation.java
package com.owlrobotics.pedropathingvisualizer.pedropathing.util;

public enum FieldRotation {
    ROTATION_90,
    ROTATION_180,
    ROTATION_270;

    public double getRotation() {
        return switch (this) {
            case ROTATION_90 -> Math.toRadians(90);
            case ROTATION_180 -> Math.toRadians(180);
            case ROTATION_270 -> Math.toRadians(270);
        };
    }
}

File: src/com/owlrobotics/pedropathingvisualizer/pedropathing/util/FieldRotation.kt
package com.owlrobotics.pedropathingvisualizer.pedropathing.util

enum class FieldRotation {
    ROTATION_90,
    ROTATION_180,
    ROTATION_270;

    val rotation: Double
        get() = when (this) {
            ROTATION_90 -> Math.toRadians(90.0)
            ROTATION_180 -> Math.toRadians(180.0)
            ROTATION_270 -> Math.toRadians(270.0)
        }
}
